src/styles/blocks/stars.css

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+}
+
+.stars--1 :nth-child(-n + 1) {
+  background-image: var(--active-star);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: var(--active-star);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: var(--active-star);
+}
+
+.stars--4 :nth-child(-n + 4) {
```

Comment:
Remove the space between `.stars--4` and `:nth-child(-n + 4)` to correctly target the first four child elements directly inside `.stars--4`.